src/python-common/ceph/deployment/service_spec.py

```diff
@@ -928,9 +928,9 @@ def __init__(self,
                  service_id: Optional[str] = None,
                  pool: Optional[str] = None,
                  trusted_ip_list: Optional[str] = None,
-                 api_port: Optional[int] = None,
-                 api_user: Optional[str] = None,
-                 api_password: Optional[str] = None,
+                 api_port: Optional[int] = 5000,
+                 api_user: Optional[str] = 'admin',
+                 api_password: Optional[str] = 'admin',
```

Comment:
there's a test `test_yaml` that seems to need a small adjustment after this
```
E       AssertionError: assert 'service_type...:1\n  - ::2\n' == 'service_type...:1\n  - ::2\n'
E         Skipping 84 identical leading characters in diff, use -v to show
E           ec:
E         +   api_password: admin
E         +   api_port: 5000
E             api_user: api_user
E             pool: pool
E             trusted_ip_list:...
E         
E         ...Full output truncated (3 lines hidden), use '-vv' to show

ceph/tests/test_service_spec.py:367: AssertionError
```

Reply:
thanks. updated!